plugins/guests/arch/cap/nfs_client.rb

```diff
@@ -0,0 +1,16 @@
+module VagrantPlugins
+  module GuestArch
+    module Cap
+      class NFSClient
+        def self.nfs_client_install(machine)
+          comm = machine.communicate
+          comm.sudo <<-EOH.gsub(/^ {12}/, '')
+            pacman -Sy --noconfirm
+            pacman -S --noconfirm nfs-utils
```

Comment:
That is not recommended, per this quote in the [Arch wiki](https://wiki.archlinux.org/index.php/Pacman#Installing_packages) and reiterated multiple times by developers:

> Warning: When installing packages in Arch, avoid refreshing the package list without upgrading the system (for example, when a package is no longer found in the official repositories). In practice, do not run pacman -Sy package_name instead of pacman -Syu package_name, as this could lead to dependency issues. See [System maintenance#Partial upgrades](https://wiki.archlinux.org/index.php/System_maintenance#Partial_upgrades_are_unsupported) are unsupported and [BBS#89328](https://bbs.archlinux.org/viewtopic.php?id=89328).